src/blocks/Slider/__stories__/data.json

```diff
@@ -107,7 +107,7 @@
     "content": {
       "type": "slider-block",
       "title": {
-        "text": "Slider",
+        "text": "Slider New",
```

Comment:
I think we can delete `New`

Reply:
fixed. Also screenshot test images were updated